cmd/plugin/builder/command/cli_compile.go

```diff
@@ -191,7 +191,6 @@ func Compile(compileArgs *PluginCompileArgs) error {
 	close(plugins)
 	close(fatalErrors)
 
-	log.BreakHard()
```

Comment:
This was meant to separate the plugin build logs from any fatal errors for some plugins.
All it printed was `=========`.
How about adding our own log: `log.Outputf("========")`?
Note that such a log will go to `stdout`, so we should only add it if we agree to send all the logs of this function to `stdout`, as suggested in the global comments.

Reply:
Replaced it with the `log.Info`.